src/logtab.cpp

```diff
@@ -39,18 +39,20 @@ void LogTab::InitTreeView(const EventListPtr events)
 
     m_bar->ShowMessage(QString("%1 events loaded").arg(QString::number(m_treeModel->rowCount())), 3000);
 
+    bool isPlainText = (events->size() > 0 && events->at(0)["k"].toString().isEmpty());
```

Comment:
Am I understanding this correctly? So isPlainText is initialized based on the first element only? What are some example logs where the ["k"] value will be empty? I don't think I've seen any of these before.